src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2179,29 +2181,31 @@ because xbuild doesn't support framework reference assemblies.
 <Target Name="_IncludeNativeSystemLibraries">
   <PropertyGroup>
     <_Assemblies>@(_ResolvedFrameworkAssemblies)</_Assemblies>
+	<_TargetInterpreterPrefix Condition=" '$(_AndroidUseInterpreter)' != 'False' ">interpreter-</_TargetInterpreterPrefix>
   </PropertyGroup>
   <ItemGroup>
     <_TargetArchitecture Include="$(_Android32bitArchitectures);$(_Android64bitArchitectures)" />
   </ItemGroup>
   <ItemGroup>
     <_TargetLibDir Include="$(MSBuildThisFileDirectory)lib\%(_TargetArchitecture.Identity)" Condition=" Exists('$(MSBuildThisFileDirectory)lib\%(_TargetArchitecture.Identity)') "/>
+	<_TargetLibInterpreterDir Include="$(MSBuildThisFileDirectory)lib\$(_TargetInterpreterPrefix)%(_TargetArchitecture.Identity)" Condition=" Exists('$(MSBuildThisFileDirectory)lib\$(_TargetInterpreterPrefix)%(_TargetArchitecture.Identity)') "/>
   </ItemGroup>
   <SplitProperty Value="$(AndroidEmbedProfilers)" Condition=" '$(AndroidEmbedProfilers)' != '' ">
     <Output TaskParameter="Output" ItemName="_EmbedProfilers" />
   </SplitProperty>
   <ItemGroup Condition=" '$(UsingAndroidNETSdk)' != 'True' ">
     <AndroidNativeLibrary Include="%(_TargetLibDir.Identity)\libsqlite3_xamarin.so" Condition="$(_Assemblies.Contains('Mono.Data.Sqlite.dll'))" />
-    <AndroidNativeLibrary Include="%(_TargetLibDir.Identity)\libMonoPosixHelper.so" Condition="$(_Assemblies.Contains('Mono.Posix.dll'))" />
```

Comment:
@grendello I'm confused.. so we not need these files when not using the interpreter?

Reply:
We do, but they come from a different directory

Reply:
So should we be including them from the old directory when not using the interpreter?

Reply:
We are - look at the diff above this section - it conditionally includes the interpreter path in `$(_TargetLibInterpreterDir)` which is otherwise identical to `$(_TargetLibDir)` and has the same value if the interpreter isn't used.